src/openlcb/If.cxx

```diff
@@ -35,9 +35,14 @@
 #include "openlcb/If.hxx"
 #include "openlcb/Convert.hxx"
 
-/// Ensures that the largest bucket in the main buffer pool is exactly the size
-/// of a GenMessage.
-const unsigned LARGEST_BUFFERPOOL_BUCKET = sizeof(Buffer<openlcb::GenMessage>);
+static constexpr unsigned BUCKET_SIZE = std::min(sizeof(Buffer<openlcb::GenMessage>), 64u + sizeof(BufferBase));
```

Comment:
should this have documentation on it?

Reply:
Actually the documentation is a few lines below ("ensures that the largest bucket is ...") but the code is broken. It should be max() not min().